client/src/api/api.js

```diff
@@ -7,7 +7,7 @@
  */
 import axios from 'axios';
 
-const usingRp = true;
+const usingRp = false;
```

Comment:
curious: can we leave this to `true`? I made this PR that sets up a proxy server for development: https://github.com/Researchify/Researchify/pull/65. With this, we can set this to true always. Let me know if it doesn't work for you

Reply:
Sure. Will get it fixed